src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/TransformRunnerIT.kt

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform
+
+import com.amazon.opendistroforelasticsearch.indexmanagement.rollup.model.dimension.DateHistogram
+import com.amazon.opendistroforelasticsearch.indexmanagement.rollup.model.dimension.Histogram
+import com.amazon.opendistroforelasticsearch.indexmanagement.rollup.model.dimension.Terms
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.model.Transform
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.model.TransformMetadata
+import com.amazon.opendistroforelasticsearch.indexmanagement.waitFor
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.schedule.IntervalSchedule
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import org.elasticsearch.common.settings.Settings
+import org.elasticsearch.index.query.TermQueryBuilder
+import org.elasticsearch.script.Script
+import org.elasticsearch.script.ScriptType
+import org.elasticsearch.search.aggregations.AggregationBuilders
+import org.elasticsearch.search.aggregations.AggregatorFactories
+import org.elasticsearch.search.aggregations.metrics.ScriptedMetricAggregationBuilder
+
+class TransformRunnerIT : TransformRestTestCase() {
+
+    fun `test transforms`() {
```

Comment:
Can't say I'm the biggest fan of smashing all the tests into a single test... 🤔

Reply:
Sure, will move to different test methods